src/universal_transfer_operator/data_providers/database/google/bigquery.py

```diff
@@ -155,6 +240,208 @@ def truncate_table(self, table):
         """Truncate table"""
         self.run_sql(f"TRUNCATE {self.get_table_qualified_name(table)}")
 
+    def is_native_autodetect_schema_available(  # skipcq: PYL-R0201
```

Comment:
Why do we have `skipcq: PYL-W0613` here?

Reply:
removed it: https://github.com/astronomer/apache-airflow-provider-transfers/pull/66/commits/fc01a4abfc6c92b41a9bc89da0bd376870348719